plugin/method-channel.go

```diff
@@ -223,3 +232,20 @@ func (m *MethodChannel) handleMethodCall(handler MethodHandler, methodName strin
 	}
 	responseSender.Send(binaryReply)
 }
+
+type PluginError struct {
```

Comment:
Can you add comment describing what `PluginError|Error|NewPluginError` are for. (make golint happy)

Reply:
@Drakirus Ok, I added some docs.